python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/class_maps.schema.yml

```diff
@@ -59,3 +59,15 @@ keys:
                   type: str
                   description: IPv6 Access-List Name.
                   convert_types: [ int ]
+            dscp:
+              type: dict
+              description: It allows match on either DSCP, ECN or both for the IPV4 and IPV6 packets.
```

Comment:
This description does not match the required key on the dscp_value.

Reply:
removed it as the key `dscp` itself looks sufficient to explain what the model is for.